server/index.js

```diff
@@ -1,18 +1,26 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path');
+const db = require('../db/index.js');
```

Comment:
This is totally nitpicking, but you don't need ".js" at the end of .js files when requiring them in Node.